### PR TITLE
mv to .md for Global_Objects/Object/assign

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Polyfill of `Object.assign` in `core-js`](https://github.com/zloirock/core-js#ecmascript-object)
+- {{jsxref("Object.defineProperties()")}}
+- [Enumerability and ownership of properties](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)
+- [Spread in object literals](/zh-CN/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md:235]: 链接未指向正确 section：

```suggestion
- [构造字面量对象时使用展开语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Spread_syntax#构造字面量对象时使用展开语法)
```

File: files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Polyfill of `Object.assign` in `core-js`](https://github.com/zloirock/core-js#ecmascript-object)
+- {{jsxref("Object.defineProperties()")}}
+- [Enumerability and ownership of properties](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md:234]:
```suggestion
- [属性的可枚举性和所有权](/zh-CN/docs/Web/JavaScript/Enumerability_and_ownership_of_properties)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md (hunk 2 above)
Comment (fuchunhui) [reply files/zh-cn/web/javascript/reference/global_objects/object/assign/index.md:234]: done

**Comment by fuchunhui** [comment]
LGTM

**Pull request merged**